ansible/roles/operator-pipeline/templates/openshift/tasks/get-bundle-related-data.yml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-bundle-related-data
```

Comment:
The name `get-bundle-related-data` is very generic and I so thought it would grab data from the bundle but that's not what it actually does. How about renaming this to `get-project-related-data` instead? Maybe it should even be merged with `certification-project-check` but I'll leave that for you to decide :smiley:

Reply:
I'll pass merging it :slightly_smiling_face:  `certification-project-check` is used separately in other pipelines.